fili-core/src/main/java/com/yahoo/bard/webservice/web/DefaultResponseFormatType.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web;
+
+import java.util.Locale;
+import java.util.Objects;
+
+/**
+ * Response format type.
+ */
+public enum DefaultResponseFormatType implements ResponseFormatType {
+    JSON,
+    CSV,
+    DEBUG,
+    JSONAPI;
```

Comment:
Are we still considering JASONAPI? I know people are only using the first 3

Reply:
We've got code for it, so we shouldn't pull it out without deprecating it for a while.